Formula/z/zimfw.rb

```diff
@@ -0,0 +1,32 @@
+class Zimfw < Formula
+  desc "Zsh plugin manager"
+  homepage "https://zimfw.sh"
+  url "https://github.com/zimfw/zimfw/releases/download/v1.17.1/zimfw.zsh.gz"
+  sha256 "c9ff0392a6b6a7edaf65a21e8f9fed6fcd40bb7f9374e52942a35cdd40bb9c0a"
+  license "MIT"
+
+  uses_from_macos "zsh" => :test
+
+  def install
+    share.install "zimfw.zsh"
+  end
+
+  def caveats
+    <<~EOS
+      To finish the set up, see:
+        https://github.com/zimfw/zimfw#homebrew
+
+      Source from #{opt_share}/zimfw.zsh in your .zshrc.
```

Comment:
Let's keep it to the Homebrew specific part, because if there is too much text people won't read it.
```suggestion
      zimfw.zsh lives in #{opt_share}/zimfw.zsh to source in your .zshrc.
```

Reply:
Sounds good. Just updated it your suggestion. Thanks for the reviews!